conda_forge_tick/utils.py

```diff
@@ -128,6 +131,77 @@ def get_default_container_run_args(tmpfs_size_mb: int = 10):
     ]
 
 
+def run_container_task(name, args, json_loads=json.loads):
+    """Run a bot task in a container.
+
+    Parameters
+    ----------
+    name : str
+        The name of the task.
+    args : List[str]
```

Comment:
Let's add the types to the signature instead. As a general principle, the rule of thumb is "be abstract in the inputs and concrete in the outputs". In this case, that means the signature could have `Iterable[str]` instead of `list[str]`.

By the way, we can use `from __future__ import annotations` and use the builtins as types (list, tuple, dict) directly with square brackets. I think that's available with Py38+